python/tvm/relay/frontend/tflite.py

```diff
@@ -705,47 +710,87 @@ def convert_div(self, op):
         # Check if the input tensor is quantized, call QNN op
         if self.is_quantized(op):
             raise tvm.error.OpNotImplemented(
-                'TFlite quantized div operator is not supported yet.')
+                'TFlite quantized DIV operator is not supported yet.')
         return self._convert_elemwise(_op.divide, op)
 
     def convert_pow(self, op):
+        """Convert TFLite POW"""
         # Check if the input tensor is quantized, call QNN op
         if self.is_quantized(op):
             raise tvm.error.OpNotImplemented(
-                'TFlite quantized pow operator is not supported yet.')
+                'TFlite quantized POW operator is not supported yet.')
         return self._convert_elemwise(_op.power, op)
 
+    def convert_squared_difference(self, op):
```

Comment:
This part shouldn’t appear in your pr if you rebase master correctly. Seems that you don’t handle it correctly and move this implementation manually. Please rebase your pr to master correctly referring this: https://docs.tvm.ai/contribute/git_howto.html